package.json

```diff
@@ -19,38 +19,44 @@
     "migrate-db": "npx sequelize-cli db:migrate"
   },
   "engines": {
-    "node": ">=14.0.0",
+    "node": ">=16.0.0",
     "yarn": ">= 1.22.5"
   },
   "keywords": [],
   "author": "Distribute Aid",
   "license": "(ISC OR GPL-3.0)",
   "dependencies": {
     "@graphql-tools/utils": "8.1.2",
-    "@sinclair/typebox": "^0.20.0",
-    "@types/graphql-depth-limit": "^1.1.2",
-    "@types/lodash": "^4.14.170",
-    "ajv": "^8.6.0",
-    "ajv-formats": "^2.1.0",
-    "apollo-server": "^3.0.0",
-    "apollo-server-express": "^3.0.0",
-    "compression": "^1.7.4",
-    "cors": "^2.8.5",
-    "dotenv": "^10.0.0",
-    "express": "^4.17.1",
-    "graphql": "^15.5.1",
-    "graphql-depth-limit": "^1.1.0",
-    "graphql-import-node": "^0.0.4",
-    "graphql-scalars": "^1.10.0",
-    "jsonwebtoken": "^8.5.1",
-    "jwks-rsa": "^2.0.3",
-    "lodash": "^4.17.21",
-    "nodemon": "^2.0.7",
-    "pg": "^8.6.0",
-    "prettier-plugin-organize-imports": "^2.2.0",
-    "reflect-metadata": "^0.1.13",
-    "sequelize": "^6.6.2",
-    "sequelize-typescript": "^2.1.0"
+    "@sinclair/typebox": "0.20.5",
+    "@types/graphql-depth-limit": "1.1.2",
+    "@types/lodash": "4.14.172",
+    "@types/supertest": "2.0.11",
+    "ajv": "8.6.2",
+    "ajv-formats": "2.1.1",
+    "apollo-server": "3.3.0",
+    "apollo-server-express": "3.3.0",
+    "bcrypt": "5.0.1",
+    "body-parser": "1.19.0",
+    "compression": "1.7.4",
+    "cookie-parser": "1.4.5",
+    "cors": "2.8.5",
+    "dotenv": "10.0.0",
+    "express": "4.17.1",
+    "graphql": "15.5.1",
+    "graphql-depth-limit": "1.1.0",
+    "graphql-import-node": "0.0.4",
+    "graphql-scalars": "1.10.0",
+    "lodash": "4.17.21",
+    "nodemon": "2.0.12",
+    "passport": "0.4.1",
+    "passport-cookie": "1.0.9",
+    "pg": "8.7.1",
+    "prettier-plugin-organize-imports": "2.3.3",
+    "reflect-metadata": "0.1.13",
+    "sequelize": "6.6.2",
+    "sequelize-typescript": "2.1.0",
+    "supertest": "6.1.6",
+    "uuid": "8.3.2"
```

Comment:
**praise:** thank you for pinning every dependency. I hope it wasn't too much of a pain.

Reply:
Only a little, not all of them are on the latest release (e.g. sequelize-typescript)